tests/jenkins/TestReleaseTagJob.groovy

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+import jenkins.tests.BuildPipelineTest
+import org.junit.Before
+import org.junit.Test
+
+class TestReleaseTagJob extends BuildPipelineTest {
+
+    @Before
+    void setUp() {
+
+        def buildManifest = 'tests/data/opensearch-build-1.1.0.yml'
+
+        def destPath = "${this.workspace}/${buildManifest}"
+
+        def artifactsPath = 'distribution-build-opensearch/1.1.0/123/linux/x64/builds/opensearch/manifest.yml'
+
+        def bucketName = 'job-s3-bucket-name'
+
+        this.registerLibTester(new DownloadFromS3LibTester(destPath, bucketName, artifactsPath, true))
+
+        this.registerLibTester(new CreateReleaseTagLibTester(buildManifest, '1.1.0'))
+
+        super.setUp()
+
+        // Variables for Release Tag Job
+        binding.setVariable('VERSION', '1.1.0')
+        binding.setVariable('BUILD_ID', '123')
+        binding.setVariable('ARTIFACT_BUCKET_NAME', bucketName)
+        binding.setVariable('BUILD_MANIFEST', buildManifest)
+
+        helper.registerAllowedMethod("checkout", [Map], {})
```

Comment:
These method registrations should be a part of `libTester` so anyone who uses this library is able to use them.

Reply:
No. Anyone uses the library will have possibility using different manifest. The method registrations here are specifically for the manifest I used.

Reply:
Since checkout and dir are part of the library, they should be a part of the lib tester

Reply:
Had a discussion with @abhinavGupta16 , and new changes are updated.